### DISCUSSION TITLE
Api limits of finding followers

### DISCUSSION THREAD

**Discussion by Britolpb** [tweepy/tweepy#2067] [Questions] [answered]
Please forgive me if this is a basic question. I am trying to build a proof of concept that analyzes who people that follow a corporate account follow. In this case, the account has 20k followers. Due to the api limits, do I have any option to request the followers of each one of those 20k users? Assuming all of them are public accounts? My issue is the api limit

**Comment by Harmon758** [comment, marked as answer]
The only option is to go through them one by one and get all the users they follow or all their followers individually.
